packages/nextjs-mf/utils/flushedChunks.ts

```diff
@@ -1,5 +1,13 @@
 import * as React from 'react';
 
+/**
+ * FlushedChunks component.
+ * This component creates script and link elements for each chunk.
+ * 
+ * @param {FlushedChunksProps} props - The properties of the component.
+ * @param {string[]} props.chunks - The chunks to be flushed.
+ * @returns {React.ReactElement} The created script and link elements.
```

Comment:
The function 'FlushedChunks' does not handle the case when the chunks array is null or undefined. This could lead to unexpected behavior or errors. Add a check to handle such cases.